src/main/java/io/lakefs/routerfs/PathMapper.java

```diff
@@ -213,6 +206,7 @@ private PathMapping findAppropriatePathMapping(Path path) {
      * PathMapping represents a pair of mapping configurations, that define source and destination prefixes to
      * be replaced.
      */
+    @Getter
```

Comment:
This annotation scares me.  IIUC, Lomboc will generate a getter method for _any_ private data member that I might add.  I.e. it changes the meaning of "private" for this class in a very strange way, and that will not be visible when adding new data members.

Auto-generated getters for single data members make sense, but I would rather have a library that does not let me do this whole-class annotation.

Reply:
I see what you mean.
As I saw it, this class would probably not contain any private fields that we wouldn't want to expose, but I can use the annotation on each field separately.